File: app/src/main/java/com/komori/yixi/main/mvp/MainPresenter.java
package com.komori.yixi.main.mvp;


import com.komori.yixi.base.RxPresenter;

import org.jetbrains.annotations.NotNull;

import javax.inject.Inject;

/**
 * Created by KomoriWu
 * on 2017-11-29.
 */


public class MainPresenter extends RxPresenter<MainContract.View> implements MainContract.
        Presenter {

    @Inject
    public MainPresenter() {

    }


    @NotNull
    @Override
    public String toStr() {
        return "mmp";
    }
}

File: app/src/main/java/com/komori/yixi/main/mvp/MainPresenter.kt
package com.komori.yixi.main.mvp


import com.komori.yixi.base.RxPresenter

import javax.inject.Inject

/**
 * Created by KomoriWu
 * on 2017-11-29.
 */


class MainPresenter @Inject
constructor() : RxPresenter<MainContract.View>(), MainContract.Presenter {

    override fun toStr() {
        view!!.showStr("sssssssssssssss")
    }
}
